render/src/main/java/org/codice/imaging/nitf/render/NitfRenderer.java

```diff
@@ -218,6 +221,44 @@ private void renderJPEG2k(final ImageSegment imageSegment, final Graphics2D targ
             }
         }
 
+        /* If R, G, B are not present, the default displayable band
+        is the LU band
+         */
+        if (imageBands.isEmpty()) {
```

Comment:
It looks like we do the same thing for LU and M bands as with RGB. Can we just add LU and M to the first switch statement with R,G and B?